pages/posts.js

```diff
@@ -0,0 +1,42 @@
+import getPosts from '../src/Services/get-posts';
+import Form from '../src/Components/Form/Form';
+import AddFormButton from '../src/Components/Form/AddFormButton';
+import { saveToLocal, loadFromLocal } from '../src/Components/lib/localStorage';
+import { useState, useEffect } from 'react';
+import dynamic from 'next/dynamic';
+
+export function getStaticProps() {
+	const data = getPosts();
+
+	return {
+		props: {
+			data,
+		},
+	};
+}
+export default function Posts({ data }) {
+	const [addPost, setAddPost] = useState(loadFromLocal('localPosts') ?? data);
+	const [formButton, setFormButton] = useState(false);
```

Comment:
Since your useState holds a boolean, it might be better to call it e.g. `isFormActive` instead of `formButton`. I think it's easier to understand what it does then.

Reply:
LGTM 🚀